frontend/src/components/activePage/NestedTables/NestedTables.tsx

```diff
@@ -23,25 +26,26 @@ export interface INestedTablesProps {
   workspaces: Array<IWorkspace>;
   templates: Array<ITemplate>;
   isManager: boolean;
+  nested: boolean;
   destroyAll: () => void;
 }
 
 type rowType = IWorkspace | ITemplate;
 
 const NestedTables: FC<INestedTablesProps> = ({ ...props }) => {
```

Comment:
Don't know if it makes sense: since it takes a `nested` prop, maybe call it `NestableTable`? What do you think?

Reply:
Sure, It would be more specific. 
Actually I don't really like the prop name "nested" because also the `InstancesTable` is nested, it just is a different component with a different shape. Basically what the `nested` prop does is it lets me decide whether to display the same table a second time or whether to display a different, yet still nested, table. Maybe I should just be explicit and call it `recurring` or something, any suggestions?